app/src/components/Copyright/index.tsx

```diff
@@ -1,21 +1,25 @@
 import React from "react";
 import { useNavigate } from "react-router-dom";
-import Link from "@mui/material/Link";
+import LinkComponent from "../Common/Link";
 
 import { Pages } from "../../types/enums";
 
 import "./copyright.scss";
 
 export default React.memo(function Copyright() {
-    const navigate = useNavigate();
+	const navigate = useNavigate();
 
-    return (
-        <div className="copyright">
-            Copyright ©
-            <Link href={Pages.aboutUs} variant="body2" onClick={() => navigate(Pages.aboutUs)}>
-                ВК-КЛОН
-            </Link>
-            {" "}{new Date().getFullYear()}.
-        </div>
-    );
-});
\ No newline at end of file
+	return (
+		<div className="copyright">
+			Copyright ©
+			<LinkComponent
+				href={Pages.aboutUs}
+				variant="body2"
```

Comment:
Also, i see that in all places of use the prop variant="body2" is passed. Maybe we can use this prop inside the LinkComponent?

Reply:
but if one day we will not use this prop?
i thought the component should be more universal

Reply:
read please about YAGNI principle
after that you can change your opinion)